### PR TITLE
#U14 feat: expertise section for tablet breakpoints

### PR DESCRIPTION
**Pull request by Yvad60**
# PR Description
Adding expertise section for tablet
# Description of tasks that were expected to be completed
- [x] Add support for tablet screens
- [x] ensure new feature don't break existing one
# How this maybe tested?

1. run `npm i` to install all required dependences 
2. run `npm run dev`

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: components/Sections/Expertise.jsx [hunk 1]
```diff
@@ -2,13 +2,14 @@ import expertiseList from "../../utils/expertiseData";
...
-        <div className="grid grid-cols-1 mt-5 gap-y-3">
+      <div className="text-xl leading-8 -tracking-0.01 md:text-2xl">
+        <h4 className=" text-muted-300">Expertise</h4>
+        <div className="grid grid-cols-1 mt-5 md:grid-cols-2 gap-y-3 gap-x-6">
           {expertiseList.map((expertise, index) => (
+            // used width of 70vw (w-70w) to intentionaly flow the text to 70% of the whole screen (wrapper margin included) but only on mobile below 400px
             <h5
               key={index}
-              className="text-xl leading-8 border-b border-muted-100 w-70w"
+              className="border-b border-muted-100 w-70w ssm:w-[18.125rem]"
```
Comment (chrisostome01) [inline components/Sections/Expertise.jsx]: what do you mean by `ssm:` and also it would be better if you put value in the tailwind config

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
Well done @Yvad60 👍, take look on these comments 📌

File: components/Sections/Expertise.jsx (hunk 1 above)
Comment (Yvad60) [reply components/Sections/Expertise.jsx]: It's a custom breakpoint added in `tailwind.config` 
I missed the arbitrary value though thanks for the catch

**Review by serapieTuyishime — APPROVED** [review]
looking good, 

I like the comments

**Review by serapieTuyishime — APPROVED** [review]
looking good, 

I like the comments

**Review by chrisostome01 — APPROVED** [review]
Well done 👍

**Pull request merged**